product/reports/170_Configuration Management - Containers/130_Pods per Ready Status.yaml

```diff
@@ -0,0 +1,47 @@
+---
+title: Pods per Ready Status
+rpt_group: Configuration Management - Containers
+rpt_type: Default
+priority: 60
+db: ContainerGroup
+cols:
+- name
+- ready_condition_status
+include: {}
+col_order:
+- name
+- ready_condition_status
+headers:
+- "# Pods per Ready Status"
+- Ready Condition Status
+conditions: 
+order: Ascending
+sortby:
+- ready_condition_status
+group: 
+graph:
+  :type: PieThreed
+  :count: 10
+  :other: true
+dims: 1
+filename: 170_Configuration Management - Containers/060_Container Groups per
+  Ready Status.yaml
```

Comment:
@nimrodshn no I mean the `filename` "170_Configuration Management - Containers/060_Container Groups per Ready Status.yaml"